ansible/group_vars/all.yml

```diff
@@ -119,6 +119,23 @@ broker_image_name: docker.io/ansibleplaybookbundle/origin-ansible-service-broker
 broker_tag: "latest"
 broker_image: "{{ broker_image_name }}:{{ broker_tag }}"
 
+# Helios Broker 
+deploy_asb: True
+deploy_helios: False
+helios_dockerhub_org: projecthelios
+helios_asb_project: aws-service-broker
+
+helios_broker_registry_url: docker.io
+helios_broker_image_name: "{{ helios_broker_registry_url }}/{{ helios_dockerhub_org }}/aws-service-broker"
+helios_broker_tag: "latest"
+helios_broker_image: "{{ helios_broker_image_name }}:{{ helios_broker_tag }}"
+
+helios_broker_template_dir: "/git/Project-Helios/deploy_scripts"
```

Comment:
We don't want to recommend or require any checkout of the deploy_scripts.

We'll want to support 2 modes of obtaining the file:
- Use a local file, similar to what we allow with the asb template
- Fetch from a URL

Reply:
will update